src/qibolab/designs/qmrs.py

```diff
@@ -0,0 +1,160 @@
+from qibo.config import log, raise_error
+
+from qibolab.designs.abstract import AbstractInstrumentDesign
+from qibolab.platforms.utils import Channel
+
+
+class QMRSDesign(AbstractInstrumentDesign):
+    """Instrument design for Quantum Machines (QM) OPXs and Rohde Schwarz local oscillators.
+
+    IPs and other instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+
+    Args:
+        address (str): Address and port for the QM OPX cluster.
+            Default is the current address for the instruments in TII lab.
+        simulation_duration (int): Duration for the simulation in ns.
+            If given the compiler simulator will be used instead of the actual hardware.
+            Default is ``None`` which falls back to the hardware.
+        cloud (bool): See :class:`qibolab.instruments.qmsim.QMSim` for details.
+            Relevant only when ``simulation_duration`` is given.
+
+    Attributes:
+        is_connected (bool): Boolean that shows whether instruments are connected.
+        opx (:class:`qibolab.instruments.qm.QMOPX`): Object used for controlling the QM OPXs.
+        local_oscillators (list): List of local oscillator objects.
+            instrument objects.
+    """
+
+    def __init__(self, address="192.168.0.1:80", simulation_duration=None, cloud=False):
```

Comment:
Are you sure about `address` default?

Reply:
This address works for the TII lab. This is the only lab that has this set of instruments and therefore will use this code, so from a practical perspective hardcoding it there is not a problem. From a code perspective though I understand your concern.

One possibility is to move this and other hardcoded values to the top of the class, as you proposed in another PR. But more generally we need a systematic way to input values from the lab (IP addresses, channel to port maps, etc.) to the designs. One possibility is to have another runcard for this, however as far as I know we did not reach a final decision yet. If you have any idea, feel free to propose.

Reply:
Done.